build/templates/tox-driver.ini.mako

```diff
@@ -0,0 +1,71 @@
+<%
+    import build.helper as helper
+
+    config = template_parameters['metadata'].config
+    module_name = config['module_name']
+    if config['supports_nitclk']:
+        nitclk_env = 'py38-{}-nitclk_wheel,'.format(module_name)
+    else:
+        nitclk_env = ''
+%>\
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox -c tox-driver.ini" from the driver directory.
```

Comment:
Shouldn't this be "tox-${module_name}.ini" ?
Can we replace "the driver directory" with something specific?

The reason is that "driver" is very ambiguous. We refer to things interchangeably as "driver" (ambiguous), Python module, Python API bindings.

We sometimes refer to the "driver runtime" as "driver".

I want to minimize ambiguity and use consistent terminology.

Reply:
After discussion with @texasaggie97 , a more descriptive name `tox-system_tests.ini` would be better.

Reply:
Changed the name